manifests/g/GiulioSorrentino/wxBriscola/0.6.3/GiulioSorrentino.wxBriscola.installer.yaml

```
@@ -4,13 +4,19 @@
 PackageIdentifier: GiulioSorrentino.wxBriscola
 PackageVersion: 0.6.3
 InstallerType: wix
-Dependencies:
-  PackageDependencies:
-  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
 ProductCode: '{76833DA3-34C1-49B0-A918-8A1825DD50FE}'
 Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/numerunix/wxBriscola-for-Windows/releases/download/0.6.3/wxBriscola-0.6.3-1-amd64.msi
-  InstallerSha256: B89C438F0602A15DBFB780E6A3EC9346E6C1B458D70FE210F705B0A2E6C66CEB
+  InstallerSha256: 57286C65E1870A1B6280808EEC7694E03084B86598E55152783786EF5A7504A2
+    Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+- Architecture: arm64
+  InstallerUrl: https://github.com/numerunix/wxBriscola-for-Windows/releases/download/0.6.3/wxBriscola-0.6.3-1-arm64.msi
+  InstallerSha256: F2013BBFEB95C9EBC6B6DAF42F90E6319F43184BB7863B3931CF72E34D81DB4C
+    Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2022.arm64

```

```suggestion
  Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
- Architecture: arm64
  InstallerUrl: https://github.com/numerunix/wxBriscola-for-Windows/releases/download/0.6.3/wxBriscola-0.6.3-1-arm64.msi
  InstallerSha256: F2013BBFEB95C9EBC6B6DAF42F90E6319F43184BB7863B3931CF72E34D81DB4C
  Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2022.arm64
```

Indentation needs to be fixed